src/components/TankTableSetupWizard.tsx

```
@@ -0,0 +1,370 @@
+  }, [vesselName, tankTables, tankGroups, mappings, saveConfiguration, saveMappingConfiguration, onComplete]);
+
+  // Check if step can be completed
+  const canProceed = useCallback(() => {
+    switch (currentStep) {
+      case 'upload':
+        return uploadResults.some(result => result.success);
+      case 'groups':
+        return tankGroups.length > 0;
+      case 'mapping':
+        // Check if all tanks are mapped
+        const totalTanks = tankData.tanks.length;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)